src/FSharpLint.Core/Rules/Hints/HintMatcher.fs

```diff
@@ -220,28 +229,29 @@ module private MatchExpression =
         | ExpressionUtilities.Identifier([ident], _), ExpressionUtilities.Identifier([opIdent], _) when opIdent.idText = "op_Equality" ->
             match arguments.FSharpCheckFileResults with
             | Some(checkFile) ->
-                fun () -> 
+                let checkSymbol () = 
                     let symbolUse =
                         checkFile.GetSymbolUseAtLocation(
-                            ident.idRange.StartLine, ident.idRange.EndColumn, "", [ident.idText])
+                            ident.idRange.StartLine, ident.idRange.EndColumn, String.Empty, [ident.idText])
 
                     match symbolUse with
                     | Some(symbolUse) ->
                         match symbolUse.Symbol with
                         | :? FSharpParameter
                         | :? FSharpField -> false
-                        | :? FSharpMemberOrFunctionOrValue as x -> not x.IsProperty
+                        | :? FSharpMemberOrFunctionOrValue as fSharpMemberOrFunctionOrValue -> not fSharpMemberOrFunctionOrValue.IsProperty
```

Comment:
@Mersho OMG don't make the mistake of going to the other extreme, this is too long! use `element` or something like this